src/components/MainCard.jsx

```diff
@@ -0,0 +1,56 @@
+import { useContext, useState } from "react";
+import { BsArrowDownUp } from "react-icons/bs";
+import styles from "./MainCard.module.css";
+import buttonStyles from "./ToggleButton.module.css";
+import Input from "./Input";
+import ToggleButton from "./ToggleButton";
+import DatePicker from "./DatePicker";
+import { TripContext } from "./TripContext";
+
+export default function MainCard() {
+  const [origin, setOrigin] = useState(() => "");
```

Comment:
```suggestion
  const [origin, setOrigin] = useState("");
```

Reply:
You can start all your states without a function, since their initial values are just simple.